ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -322,6 +332,21 @@ public func |> <T, E, X>(producer: SignalProducer<T, E>, @noescape transform: Si
 	return transform(producer)
 }
 
+/// Applies a SignalProducer operator to a Property.
+///
+/// Example:
+///
+/// 	intProperty
+/// 	|> startOn(UIScheduler())
```

Comment:
I know this is not really a useful application for an example, but I didn't find an operator that is __only__ available as a Signal Operator AND is more useful than this... :disappointed_relieved: